recipes/shs-libcxi/build.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+set -ex
+
+sed -i '/AC_INIT/a AC_CONFIG_MACRO_DIR([m4])' configure.ac
```

Comment:
Can you add comments as to why these are needed and emphasize why it is not appropriate to upstream these changes. 

Pointing to an existing upstream issue is fine too.

Reply:
Actually only the one sed about the path for csr_defs.json is really needed
Removed the other ones